fud2/tests/runt/emit-json/calyx-to-verilog-round-trip.sh

```diff
@@ -0,0 +1,7 @@
+found="$(fud2 fud2/tests/runt/emit-json/calyx.futil --to verilog -m json-plan | fud2 fud2/tests/runt/emit-json/calyx.futil --to verilog --planner json)"
+expected="$(fud2 fud2/tests/runt/emit-json/calyx.futil --to verilog)"
```

Comment:
This is essentially a "differential testing" setup: we want two different commands to produce the same output, but we don't care (i.e., we don't check into git) what that output is. I guess maybe I suggest that, for a Runt-based snapshot test setup, the more natural thing to do here is to just check in the snapshot and instruct both tests to use the same expected output file.

As in, you'd run these two commands as two separate tests, and the expected output file for both would be the Verilog output.